spec/reek/cli/options_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../spec_helper'
+
+require_relative '../../../lib/reek/cli/options'
+
+describe Reek::CLI::Options do
+  describe '#initialize' do
+    let(:argv) { [] }
```

Comment:
Can you either inline this `let` or make `CLI::Options.new`’s parameter optional? I understand the need for DRY-ness, but I think this makes the individual tests a bit less readable (you have to jump to the `let` just to see that it’s an empty `Array`).